7.3/cli/Dockerfile

```diff
@@ -47,7 +47,11 @@ RUN { \
 		echo 'opcache.max_accelerated_files=10000'; \
 		echo 'opcache.validate_timestamps=0'; \
 		echo 'opcache.fast_shutdown=1'; \
-	} >> /usr/local/etc/php/conf.d/docker-php-ext-opcache.ini;
+	} >> /usr/local/etc/php/conf.d/docker-php-ext-opcache.ini; \
+	\
+	{ \
+		echo 'memory_limit=512M'; \
+	} >> /usr/local/etc/php/conf.d/custom.ini;
```

Comment:
The solution looks good in terms of setting the memory limit. One thought is that this file could be named zcustom.ini to come last. That means we know any directives will override anything in any of the other files in conf.d